sawb_custom_hide_vat_for_individual/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "SAWB Custom : Hide VAT for Individual",
+    "summary": """
+        Hide the partner's VAT field if the partner is an individual.""",
+    "version": "16.0.1.0.0",
+    "category": "Contact",
+    "website": "https://coopiteasy.be",
+    "author": "Coop IT Easy SC",
+    "maintainers": ["victor-champonnois"],
+    "license": "AGPL-3",
+    "application": False,
+    "depends": [],
+    "excludes": [],
+    "data": ["views/res_partner_view.xml"],
+    "demo": [],
+    "qweb": [],
```

Comment:
```suggestion
    "data": [
        "views/res_partner_view.xml",
    ],
```
please [avoid empty keys](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6) and use one line per list element like in [the example manifest](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py).

Reply:
Noted, thanks !